README.md

```diff
@@ -15,6 +15,7 @@ It is similar to [Netflix/denominator](https://github.com/Netflix/denominator).
 - [Table of Contents](#table-of-contents)
 - [Getting started](#getting-started)
   - [Workspace](#workspace)
+    - [Installing a specific commit SHA](#installing-a-specific-commit-sha)
```

Comment:
Looks like my VS code extension added this without me noticing

Reply:
Guess it's not wrong.